### PR TITLE
15499 Firebase deployment and convert configmap and keycloak to .env

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* [15499](https://github.com/bcgov/entity/issues/15499)

*Description of changes:*
1. Enable firebase deployment;
2. Convert configuration.json to .env;
3. Convert keycloak.json to .env;
4. Remove PaymentSystemAlert and replace with banner-text;

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: Makefile
Comment (severinbeauvais) [inline Makefile:63]: So we still use the OCP tools project?

File: src/store/Configuration/actions.ts
Comment (severinbeauvais) [inline src/store/Configuration/actions.ts:12]: I'll let @jonathan-longe speak to whether Entities Team needs a follow-up ticket to clean this up further (ie, maybe move it elsewhere?).

LGTM Patrick - nice job 👍

**Comment by severinbeauvais** [comment]
NB: please ignore the remaining SonarCloud Code Analysis issues -- I'm working on them in my current ticket in this repo.

**Review by JazzarKarim — APPROVED** [review]
Looks good to me!

File: Makefile
Comment (pwei1018) [reply Makefile:63]: Yes, we still deploy the code to Openshift.

**Comment by pwei1018** [comment]
> LGTM. Please get some other eyes on this before merging.
> 
> Also -- there is active development in this repo currently, including some blocked/blocking tickets. What impact will these changes have on:
> 
> * CI?
> * CD?
> * deploys to Test?
> * deploys to Prod?

It shouldn't impact anything for now. When you deploy to Test and Prod, please let me know. I need to watch the CD running status.

File: src/store/Configuration/actions.ts
Comment (pwei1018) [reply src/store/Configuration/actions.ts:12]: Yes, both.

**Comment by severinbeauvais** [comment]
@pwei1018 Should be above checks pass?

**Review by severinbeauvais — APPROVED** [review]
You have my approval, but please fix broken checks before merging,

**Pull request merged**
